app/models/user.rb

```diff
@@ -0,0 +1,20 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+  before_save{email.downcase}
+
+  validates :name,
+            presence: true,
+            length: {maximum: Settings.digit.length_50}
+
+  validates :email,
+            presence: true,
+            length: {maximum: Settings.digit.length_255},
+            format: {with: VALID_EMAIL_REGEX},
+            uniqueness: {case_sensitive: false}
+
+  has_secure_password
+
+  validates :password,
+            length: {minimum: Settings.digit.length_6}
```

Comment:
```suggestion
  validates :password,
            length: {minimum: Settings.digit.length_6}
            
  has_secure_password
```
ri cho nó đỡ cô đơn, tự dưng mớ validate chen thèn has secure vào bé